src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -11,46 +14,89 @@ export interface InfoViewerItem {
     value: React.ReactNode;
 }
 
+export interface InfoViewerHeaderLabel {
+    value: React.ReactNode;
+    icon?: IconData;
+    theme?: LabelProps['theme'];
+}
+
 export interface InfoViewerProps {
     title?: React.ReactNode;
+    titleSuffix?: React.ReactNode;
+    separator?: React.ReactNode;
     info?: InfoViewerItem[];
     dots?: boolean;
     size?: 's';
     variant?: 'default' | 'small';
     className?: string;
     multilineLabels?: boolean;
     renderEmptyState?: (props?: Pick<InfoViewerProps, 'title' | 'size'>) => React.ReactNode;
+    headerLabels?: InfoViewerHeaderLabel[];
 }
 
 const b = cn('info-viewer');
 
 export const InfoViewer = ({
```

Comment:
`InfoViewer` is old component, we want to get rid of it in turn to `DefinitionList` from `@gravity-ui/uikit`. Please, do not improve it, move `PDiskPopup` to `DefinitionList` instead.